dist/package.sh

```diff
@@ -78,12 +83,23 @@ targets=(
     sdp/sdp.js
 )
 
+
 for target in "${targets[@]}"; do
-    install -D "node_modules/$target" "$DIR/node_modules/$target"
+
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        ditto "node_modules/$target" "$DIR/node_modules/$target"
+    else
+        install -D "node_modules/$target" "$DIR/node_modules/$target"
+    fi
 done
 
+
 echo "+ Update version number..."
-sed -i "s/\[\[VERSION\]\]/${VERSION}/g" $DIR/index.html $DIR/troubleshoot/index.html $DIR/dist/app.js $DIR/version.txt
+if [[ "$OSTYPE" == "darwin"* ]]; then
+        sed -i'' -e "s/\[\[VERSION\]\]/${VERSION}/g" $DIR/index.html $DIR/troubleshoot/index.html $DIR/dist/app.js $DIR/version.txt
+    else
+        sed -i "s/\[\[VERSION\]\]/${VERSION}/g" $DIR/index.html $DIR/troubleshoot/index.html $DIR/dist/app.js $DIR/version.txt
```

Comment:
Same thing here. But careful, you need to add a space between the `i` and the `''`, otherwise it won't work!

Reply:
Also fixed in https://github.com/threema-ch/threema-web/commit/95883b815973c4c91681315260d25d3ddbea1d5a